internal/cmd/plugin/list/cmd.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright The CloudNativePG Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package list
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cloudnative-pg/cloudnative-pg/internal/cmd/plugin"
+)
+
+// NewCmd create the new "status" subcommand
+func NewCmd() *cobra.Command {
+	var allNamespaces bool
+
+	listCmd := &cobra.Command{
+		Use:   "list",
+		Short: "List all created PostgreSQL cluster",
+		// Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			// clusterName := args[0]
+
+			output, _ := cmd.Flags().GetString("output")
+
+			return List(ctx, allNamespaces, plugin.OutputFormat(output))
+		},
+	}
+
+	listCmd.PersistentFlags().BoolVarP(&allNamespaces,
```

Comment:
What about a label selector too?

Reply:
I hadn't thought of that. But I can add that if you like.

Reply:
I just had a look, so something like `kubectl-cnpg list --selector=app=tas-core`? Not sure how to do this with Cobra. :D I tried to look for an example, but don't see anything in this repo yet.

Reply:
I think done. :D